lib/prompts/checkbox.js

```diff
@@ -96,7 +96,8 @@ Prompt.prototype.render = function (error) {
     message += chalk.cyan( this.selection.join(", ") );
   } else {
     var choicesStr = renderChoices(this.opt.choices, this.pointer);
-    message += "\n" + this.paginator.paginate(choicesStr, this.pointer, this.opt.pageSize);
+    var indexPosition = this.opt.choices.choices.indexOf(this.opt.choices.getChoice(this.pointer));
```

Comment:
Can you add an `indexOf` function on the `Choices` collection object rather than accessing inner properties of the object (`choices.indexOf` rather than `choices.choices.indexOf`). This is already done for other `Array.prototype` methods here: https://github.com/SBoudrias/Inquirer.js/blob/master/lib/objects/choices.js#L98